var/spack/repos/builtin/packages/py-jedi/package.py

```diff
@@ -34,13 +35,16 @@ class PyJedi(PythonPackage):
 
     depends_on('py-setuptools', type=('build', 'run'))
 
-    depends_on('python@2.6:2.8,3.2:', type=('build', 'run'), when='@0.9.0')
-    depends_on('python@2.6:2.8,3.3:', type=('build', 'run'), when='@0.10.0')
-    depends_on('python@2.7:2.8,3.4:', type=('build', 'run'), when='@0.13.3')
-    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'), when='@0.17.2')
-    depends_on('python@3.6:', type=('build', 'run'), when='@0.18.0')
+    depends_on('python@2.6:2.8,3.2:', type=('build', 'run'), when='@0.9.0:')
+    depends_on('python@2.6:2.8,3.3:', type=('build', 'run'), when='@0.10.0:')
+    depends_on('python@2.7:2.8,3.3:', type=('build', 'run'), when='@0.12.0:')
+    depends_on('python@2.7:2.8,3.4:', type=('build', 'run'), when='@0.13.3:')
+    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'), when='@0.17.0:')
+    depends_on('python@3.6:', type=('build', 'run'), when='@0.18.0:')
 
     depends_on('py-parso@0.2.0:', type=('build', 'run'), when='@0.12.0')
     depends_on('py-parso@0.3.0:', type=('build', 'run'), when='@0.12.1:0.14.0')
-    depends_on('py-parso@0.7.0:0.7', type=('build', 'run'), when='@0.17.2')
-    depends_on('py-parso@0.8.0:0.8', type=('build', 'run'), when='@0.18.0')
+    depends_on('py-parso@0.5.0:', type=('build', 'run'), when='@0.14.1:0.15.1')
+    depends_on('py-parso@0.5.2:', type=('build', 'run'), when='@0.15.1:0.16')
```

Comment:
Version ranges are inclusive, so it's unclear which of these 0.15.1 belongs to. Also, many of these can be simplified like:
```python
depends_on('py-parso@0.7', when='@0.17', ...)
```

Reply:
I have fixed the version.

The reason why I might have overspecified the versions here, is that I made these changes right after encountering #29170 and was not sure about the syntax anymore :-).

Reply:
I've been using Spack for 7 years and I've never been sure about the syntax 🤣